components/dashboard/src/Login.tsx

```diff
@@ -48,36 +69,43 @@ export function Login() {
+                <div className="flex flex-col px-8 my-auto ml-auto">
+                    <div className="mb-12">
                         <img src={gitpod} className="h-8" />
                     </div>
-                    <div>
-                        <h1 className="xl:text-7xl text-5xl">Save Time<br /> with Prebuilds</h1>
+                    <div className="mb-10">
+                        <h1 className="text-5xl mb-3">Welcome to Gitpod</h1>
+                        <div className="text-gray-400 text-lg">
+                            Spin up fresh, automated dev environments for each task in the cloud, in seconds.
+                        </div>
                     </div>
-                    <div className="text-gray-400 text-lg">
-                        Gitpod continuously builds your git branches like a CI server. This means no more waiting for dependencies to be downloaded and builds to finish. <a className="underline underline-thickness-thin underline-offset-small hover:text-gray-600" href="https://www.gitpod.io/docs/prebuilds/" target="gitpod-docs">Learn more about Prebuilds</a>
+                    <div className="flex mb-10">
+                        <Item icon={code} iconSize="16" text="Always Ready&#x2011;To&#x2011;Code"/>
+                        <Item icon={customize} text="Personalize your Workspace"/>
+                        <Item icon={automate} text="Automate Your Development Setup"/>
                     </div>
-                    <div>
-                        <img src={terminal} className="h-64 -ml-8" />
+                    <div className="flex">
+                        <Item icon={prebuild} text="Continuously Prebuild Your Project"/>
+                        <Item icon={collaborate} text="Collaborate With Your Team"/>
+                        <Item icon={fresh} text="Fresh Workspace For Each New Task"/>
                     </div>
                 </div>
             </div>
-        </div>
-        <div id="login-section" className="flex-grow flex lg:w-1/2 w-full">
-            <div id="login-section-column" className="flex-grow max-w-2xl flex flex-col h-100 mx-auto lg:mx-0">
+        </div>: null}
+        <div id="login-section" className={"flex-grow flex w-full" + (showWelcome ? " lg:w-1/2" : "")}>
+            <div id="login-section-column" className={"flex-grow max-w-2xl flex flex-col h-100 mx-auto" + (showWelcome ? " lg:my-0" : "")}>
                 <div className="flex-grow h-100 flex flex-row items-center justify-center" >
                     <div className="rounded-xl px-10 py-10 mx-auto">
                         <div className="mx-auto pb-8">
                             <img src={gitpodIcon} className="h-16 mx-auto" />
                         </div>
                         <div className="mx-auto text-center pb-8 space-y-2">
-                            <h1 className="text-3xl">Log in to Gitpod</h1>
+                            <h1 className="text-3xl">Log in</h1>
```

Comment:
**question**: When the left column is abent, we don't have a single reference of the Gitpod term, right? It should be fine for now.

Reply:
Yes, we do not. I will make it conditional.